etna/transforms/outliers/base.py

```diff
@@ -78,6 +85,15 @@ def fit(self, ts: TSDataset) -> "OutliersTransform":
         :
             The fitted transform instance.
         """
+        if self.ignore_flag_column is not None:
+            if self.ignore_flag_column not in ts.columns.get_level_values(1).values.tolist():
```

Comment:
Same about level name. Also `values.tolist()` is redundant as we do index lookup